### PR TITLE
Defect/facet drag n drop

### PR DESCRIPTION
**Pull request by AndrewCochrane24**
#### Checklist

* [x] The contents of this PR are mine to submit. No third party code or other material is being committed.
* [x] New third party dependencies (if any) are linked via `package.json`. Third party dependencies are licensed under one of the following: Apache, MIT, BSD, Mozilla Public License, Eclipse Public License, or Oracle Binary Code License.
* [x] The code in this PR does not contain export-restricted encryption algorithms.

#### Ticket / Issue

Facet drag and drop feature didn't work, either dragging across one row or multiple.

#### Description of Proposed Changes

Added solution that addresses this.

#### Documentation CI URL

https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_defect--facet-drag-n-drop

#### Downstream Build(s)

https://jenkins.swinfra.net/job/SEPG/job/caf/view/Developer/job/caf%7Eux-aspects-micro-focus%7Edefect%20%E2%81%84%20facet-drag-n-drop%7ECI/

### CODE REVIEW

**Review by JonathanToddMF — COMMENTED** [review]
Hey @AndrewCochrane24 looks like we are getting some linting errors in the build

https://jenkins.swinfra.net/job/SEPG/job/UXAspects/job/UXAspects~UXAspects~defect%20%E2%81%84%20facet-drag-n-drop~CI/

**Review by JonathanToddMF — CHANGES_REQUESTED** [review]
Hey @AndrewCochrane24 great work on this, the only slight thing I am noticing is this slightly odd styling when grabbing a facet and moving the mouse slightly.

[Image from Gyazo](https://gyazo.com/91434c209bbbbe01aa4a589378433e71)

You may also notice I added a downstream build to the PR, this builds after the open source builds and runs the build for the internal version of the library. The internal library imports a lot of the components from the open source library (you have to manually start it, I can kick it off for you if you push any updates). The facet-container works and looks well in there too. Just with the same odd styling issue mentioned above.

https://pages.github.houston.softwaregrp.net/Inner-Source-Incubating/universal-codesandbox/#/editor?projectId=fc907941-8f20-421a-bc61-a2c9a6786191

**Comment by AndrewCochrane24** [comment]
> Hey @AndrewCochrane24 great work on this, the only slight thing I am noticing is this slightly odd styling when grabbing a facet and moving the mouse slightly.
> 
> [Image from Gyazo](https://gyazo.com/91434c209bbbbe01aa4a589378433e71) [ Image from Gyazo ](https://gyazo.com/91434c209bbbbe01aa4a589378433e71) [ ](https://gyazo.com/91434c209bbbbe01aa4a589378433e71)
> 
> You may also notice I added a downstream build to the PR, this builds after the open source builds and runs the build for the internal version of the library. The internal library imports a lot of the components from the open source library (you have to manually start it, I can kick it off for you if you push any updates). The facet-container works and looks well in there too. Just with the same odd styling issue mentioned above.
> 
> https://pages.github.houston.softwaregrp.net/Inner-Source-Incubating/universal-codesandbox/#/editor?projectId=fc907941-8f20-421a-bc61-a2c9a6786191

Sorry for the late reply Jonathan, I'll fix this now.

**Review by JonathanToddMF — APPROVED** [review]
see below

File: src/components/facets/facet-container.component.html [hunk 1]
```diff
@@ -14,39 +14,62 @@
...
 
-            <!-- Display Remove Icon -->
-            <button type="button"
-                uxFocusIndicator
-                class="facet-selected-remove-btn"
-                [attr.aria-label]="deselectFacetAriaLabel"
-                (click)="deselectFacet(facet, tag)">
+                    <!-- <div *cdkDragPlaceholder class="facet-drag-placeholder">
+                        <span class="facet-selected-tag-label">{{ facet.title }}</span>
+                    </div> -->
```
Comment (JonathanToddMF) [inline src/components/facets/facet-container.component.html:59]: Remove the commented out code

File: src/components/facets/facet-container.component.less [hunk 2]
```diff
@@ -93,15 +93,30 @@ ux-facet-container {
     }
 }
 
-.facet-selected-tag.cdk-drag-preview {
+.facet-drag-item.cdk-drag-preview {
+    // background-color: orange;
```
Comment (JonathanToddMF) [inline src/components/facets/facet-container.component.less:97]: here too

**Review by JonathanToddMF — CHANGES_REQUESTED** [review]
Sorry, everything looks good. Just noticed some commented out code that can be removed

File: src/components/facets/facet-container.component.less (hunk 2 above)
Comment (AndrewCochrane24) [reply src/components/facets/facet-container.component.less]: Done.

File: src/components/facets/facet-container.component.html (hunk 1 above)
Comment (AndrewCochrane24) [reply src/components/facets/facet-container.component.html]: Done.

**Pull request merged**